### PR TITLE
Show Incorporation Number as `Pending` if file_type is incorporationApplication and status is PAID #4658

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#4658

*Description of changes:*
  - Show Incorporation Number as `Pending` if file_type is incorporationApplication and status is PAID

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: queue_services/entity-emailer/src/entity_emailer/email_templates/common/business-info.html [hunk 1]
```diff
@@ -19,14 +19,12 @@
 
   <tr>
     <td class="name">Incorporation Number:</td>
-    {% if header.status == 'PAID' %}
+    {% if header.status == 'PAID' and filing_type == 'incorporationApplication'
+    %}
```
Comment (severinbeauvais) [inline queue_services/entity-emailer/src/entity_emailer/email_templates/common/business-info.html:23]: I think there is confusion here. The bug is that the Incorporation Number displays as "Pending"; instead the actual incorporation number should be displayed.

Comment (vysakh-menon-aot) [reply queue_services/entity-emailer/src/entity_emailer/email_templates/common/business-info.html:23]: It shows "Pending" only when the filing_type is incorporationApplication and status is Paid (at this point we don't have  Incorporation Number). In all other cases (including COD, COA) it shows Incorporation Number. Isn't that the expectation?

Comment (severinbeauvais) [reply queue_services/entity-emailer/src/entity_emailer/email_templates/common/business-info.html:23]: The description of the bug refers to a Director Change (aka COD) filing confirmation email. In this case the business is already incorporated and has an incorporation number.
I am not sure that this code change is necessary.

Comment (thorwolpert) [reply queue_services/entity-emailer/src/entity_emailer/email_templates/common/business-info.html:23]: We should use the term **identifier**, as in our model, taken from json-ld/schema.org
Using too many different terms, will make it less clear.

Comment (vysakh-menon-aot) [reply queue_services/entity-emailer/src/entity_emailer/email_templates/common/business-info.html:23]: @severinbeauvais at this case the filing_type will be `changeOfDirectors`, so it shows incorporation number.

Comment (vysakh-menon-aot) [reply queue_services/entity-emailer/src/entity_emailer/email_templates/common/business-info.html:23]: @thorwolpert I am not aware of json-ld/schema.org, can you explain that?

Comment (severinbeauvais) [reply queue_services/entity-emailer/src/entity_emailer/email_templates/common/business-info.html:23]: Vysakh wrote:
> at this case the filing_type will be changeOfDirectors, so it shows incorporation number

Aaah, OK, now I get it: you're saying that this code inadvertently displayed "Pending" for all filings (including COD), but this code should apply only to IAs.

Comment (severinbeauvais) [reply queue_services/entity-emailer/src/entity_emailer/email_templates/common/business-info.html:23]: Please create a new conversation for the naming/schema comment, and then resolve THIS conversation.

**Pull request merged**